src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -64,6 +109,10 @@ public void refreshList(String filter) {
                 .filter(pi -> pi.filter(query))
                 .sorted(Comparator.comparing(pi -> pi.username))
                 .forEach(playersModel::addElement);
+        if (nearbyPlayerList != null)
```

Comment:
This can never be false. nearbyPlayerList is on the initializer and is final. you can just remove the if